### DISCUSSION TITLE
Run workflow button is not working

### DISCUSSION THREAD

**Discussion by raginiDazn** [community/community#142112] [Pull Requests]
In CD pipeline, if i am trying run workflow it's not working

**Comment by yolorabbit777** [comment]
Hi @raginiDazn ,

Hope you are doing well.

I think your workflow yaml file is not correct.
CD pipelines are typically defined in YAML configuration files. A small typo or syntax error can prevent the workflow from running.

Ensure that your configuration (e.g., .github/workflows/your_workflow.yml, .gitlab-ci.yml, etc.) is valid.
Many platforms provide a validation tool or linter to check for issues in the YAML file.

Best,
@yolorabbit

**Reply by raginiDazn** [reply]
No, it was working in web but not in app
